isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -1411,4 +1363,231 @@ QString BundleObservation::formatBundleOutputString(bool errorPropagation, bool
+      measureCamera->GroundMap()->GetdXYdTOrientation(SpiceRotation::WRT_Declination, 1,
+                                                      &coeffTarget(0, index),
+                                                      &coeffTarget(1, index));
+      index++;
+    }
+
+    if (bundleSettings->solveTargetBody() && bundleSettings->solvePM()) {
+      measureCamera->GroundMap()->GetdXYdTOrientation(SpiceRotation::WRT_Twist, 0,
+                                                      &coeffTarget(0, index),
+                                                      &coeffTarget(1, index));
+      index++;
+    }
+
+    if (bundleSettings->solveTargetBody() && bundleSettings->solvePMVelocity()) {
+      measureCamera->GroundMap()->GetdXYdTOrientation(SpiceRotation::WRT_Twist, 1,
+                                                      &coeffTarget(0, index),
+                                                      &coeffTarget(1, index));
+      index++;
+    }
+
+    if (bundleSettings->solveTargetBody() && bundleTargetBody->solveMeanRadius()) {
+      std::vector<double> lookBWRTMeanRadius =
+          measureCamera->GroundMap()->MeanRadiusPartial(surfacePoint,
+                                                        bundleTargetBody->meanRadius());
+
+      measureCamera->GroundMap()->GetdXYdPoint(lookBWRTMeanRadius, &coeffTarget(0, index),
+                                               &coeffTarget(1, index));
+      index++;
+    }
+
+    if (bundleSettings->solveTargetBody() && bundleTargetBody->solveTriaxialRadii()) {
+
+      std::vector<double> lookBWRTRadiusA =
+          measureCamera->GroundMap()->EllipsoidPartial(surfacePoint,
+                                                       CameraGroundMap::WRT_MajorAxis);
+
+      measureCamera->GroundMap()->GetdXYdPoint(lookBWRTRadiusA, &coeffTarget(0, index),
+                                               &coeffTarget(1, index));
+      index++;
+
+      std::vector<double> lookBWRTRadiusB =
+          measureCamera->GroundMap()->EllipsoidPartial(surfacePoint,
+                                                       CameraGroundMap::WRT_MinorAxis);
+
+      measureCamera->GroundMap()->GetdXYdPoint(lookBWRTRadiusB, &coeffTarget(0, index),
+                                               &coeffTarget(1, index));
+      index++;
+
+      std::vector<double> lookBWRTRadiusC =
+          measureCamera->GroundMap()->EllipsoidPartial(surfacePoint,
+                                                       CameraGroundMap::WRT_PolarAxis);
+
+      measureCamera->GroundMap()->GetdXYdPoint(lookBWRTRadiusC, &coeffTarget(0, index),
+                                               &coeffTarget(1, index));
+      index++;
+    }
+
+    return true;
+  }
+
+
+  bool BundleObservation::computeImagePartials(matrix<double> &coeffImage, BundleMeasure &measure) {
+    coeffImage.clear(); 
+
+    const BundleObservationSolveSettingsQsp observationSolveSettings =
+        measure.observationSolveSettings();
```

Comment:
The observation stores its own solve settings, you should use them, not the ones from the measure. We should probably remove them from the measure.

Reply:
Agreed. I think separate PR to remove the observations from the measure?